go.mod

```diff
@@ -3,26 +3,26 @@ module github.com/kubernetes-csi/external-attacher
 go 1.18
 
 require (
-	github.com/container-storage-interface/spec v1.6.0
+	github.com/container-storage-interface/spec v1.7.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/evanphx/json-patch v5.6.0+incompatible
 	github.com/golang/mock v1.6.0
 	github.com/golang/protobuf v1.5.2
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v4 v4.0.2
 	google.golang.org/grpc v1.50.1
-	k8s.io/api v0.25.3
-	k8s.io/apimachinery v0.25.3
-	k8s.io/client-go v1.5.2
-	k8s.io/csi-translation-lib v0.25.3
+	k8s.io/api v0.25.4
+	k8s.io/apimachinery v0.25.4
+	k8s.io/client-go v0.25.4
+	k8s.io/csi-translation-lib v0.25.4
```

Comment:
Can you remove all those "replace" lines?  Those redirect it back to 0.25.0.

Reply:
@xing-yang sure, done.. ptal. thanks